### PR TITLE
180 create static conference page

### PR DESCRIPTION
**Pull request by mtheoryx**
This is a very rough work in progress.

We DO have:

Schedule data, ready for styling and additional information
Speaker data, ready for styling and additional information
A site-wide banner across all pages, that links to the conference page.

Much more is needed, but the data is there, the page is there, the nav is there.

Please review with me before merging.

### CODE REVIEW

File: src/components/conference/lcc3-schedule.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
+[
+  {
+    "est": "7:00",
+    "utc": "13:00",
+    "name": "Johanna Koester",
+    "email": "koester@us.ibm.com",
```
Comment (csharpfritz) [inline src/components/conference/lcc3-schedule.json:6]: Let's exclude the speakers email addresses form the website.  They should not be published

**Comment by csharpfritz** [comment]
Needs formatting.... I'll apply some basics for the speakers and sessions

**Comment by csharpfritz** [comment]
I've added some boxes around the sessions... and they could use some color

The session block on the schedule should have a speaker photo and the speaker name should be a link to their bio and session lower on the page

File: src/components/conference/lcc3-schedule.json (hunk 1 above)
Comment (mtheoryx) [reply src/components/conference/lcc3-schedule.json]: Roger that.

**Comment by csharpfritz** [comment]
Let's get this deployed and circle back in a separate PR with speaker images and sponsors

**Comment by SanaRinomi** [comment]
I made changes to the CSS and HTML to match more the rest of the site as I found the previous look to be too barebones.

Changes mainly done to the event and speaker sections.

Other recommended changes that can be left for another time is swap the sections of articles and vice-versa, since HTML5 articles are supposed to indicate that the information within it has enough info to stand on its own without external info, while a section is meant to represent only a section of data within a greater element.

**Review by SanaRinomi — APPROVED** [review]
I made changes to the CSS and HTML to match more the rest of the site as I found the previous look to be too barebones.

Changes mainly done to the event and speaker sections.

Other recommended changes that can be left for another time is swap the sections of articles and vice-versa, since HTML5 articles are supposed to indicate that the information within it has enough info to stand on its own without external info, while a section is meant to represent only a section of data within a greater element.

**Pull request merged**